classification/classification_network.py

```diff
@@ -21,65 +21,41 @@
 # ----------------------------------------------------------------------
 """
 The methods here are a factory to create a classification network:
-  encoder -> SP -> TM -> (UP) -> classifier
+  encoder -> (SP) -> (TM) -> (UP) -> classifier
```

Comment:
If we can specify the regions we should say so: 

    """
    The methods here are a factory to create a classification network of any of sensor, SP, TM, UP, and classifier regions.
    """